recipes-core/custom-nftables-rules/files/nftables-production.conf

```diff
@@ -0,0 +1,75 @@
+
+table inet filter {
+    chain input {
+        type filter hook input priority 0; policy drop
+
+        # Allow established/related connections
+        ct state established,related accept
+
+        # Allow only builder endpoint connections to these ports
+        ip saddr $BUILDER_ENDPOINT tcp dport 9000 accept comment "FB-operated Consensus Client P2P"
+        ip saddr $BUILDER_ENDPOINT tcp dport 30303 accept comment "FB-operated Execution Client P2P"
+        ip saddr $BUILDER_ENDPOINT tcp dport 27017 accept comment "Authenticated Searcher Input Channel"
+        ip saddr $BUILDER_ENDPOINT tcp dport 8547 accept comment "Builder State Diff Stream"
+
+        # Allow traffic from searcher namespace EXCEPT to log collector
+        iifname "veth0" ip daddr != $SEARCHER_LOG_HOST accept comment "Searcher network interface"
+
+        # Block SSH in production mode
+        tcp dport 22 drop comment "SSH blocked in production"
+    }
+
+    
+    # searcher namespace       root namespace        internet/other hosts 
+    # [veth1]  <---> [veth0] <---> [eth0] <---> [external]
+    chain forward {
+        type filter hook forward priority 0; policy drop
+        
+        # Allow searcher to access builder endpoint only
+        iifname "veth0" ip daddr $BUILDER_ENDPOINT tcp dport 9000 accept comment "FB-operated Consensus Client P2P"
+        iifname "veth0" ip daddr $BUILDER_ENDPOINT tcp dport 30303 accept comment "FB-operated Execution Client P2P"
+        iifname "veth0" ip daddr $BUILDER_ENDPOINT tcp dport 8645 accept comment "Builder RPC"
+        
+        # Explicitly block searcher's access to log collector
+        iifname "veth0" ip daddr $SEARCHER_LOG_HOST drop comment "Block direct log collector access"
+    }
+
+    #                 Searcher Namespace        Root Namespace            External
+    #                 [veth1 10.0.0.2]         [veth0 10.0.0.1]           [eth0]
+    #        |                                       |                                |
+    #        |                                       |                                |
+    #        |           forward chain               |              forward chain     |
+    #        |         <--------------->             |            <--------------->   |
+    #        |                                       |                                |
+    #        |           input chain                 |               input chain      |
+    #        |         <-------------                |            <-------------      |
+    #        |                                       |                                |
+    #        |          output chain                 |               output chain     |
+    #        |         ------------->                |            ------------->      |
+    
+    chain output {
+        type filter hook output priority 0; policy drop
+
+        # Allow established/related connections
+        ct state established,related accept
```

Comment:
hm, if a searcher establishes a connection during maintenance to their own server, wouldn't this rule let them continue sending stuff? Or are you maybe dropping the established connections on transition to production?